scripts/test

```diff
@@ -7,33 +7,21 @@ if [[ -n "${OSM_EXTRACTION_DEBUG}" ]]; then
 fi
 
 function usage() {
-    echo -n "Usage: $(basename "$0") [OPTION]
-
-Run linters and tests.
-
-Options:
-    -h --help       Display this help text
-    --git           Check git commit titles
-"
+    echo -n \
+         "Usage: $(basename "$0")
+Run linters and tests."
 }
 
-function app_tests() {
-    # TODO: Run tests
-    true;
-}
+if [ "${BASH_SOURCE[0]}" = "${0}" ]
+then
+    if [ "${1:-}" = "--help" ]
+    then
+        usage
+    else
+        pushd ./src
 
-function git_tests() {
-    # Fail build if any commit title in this branch contains these words
-    if git log --oneline develop... | grep -wiE "fixup|squash|wip"; then
-        echo "ERROR: Please squash all changes before merging."
-        exit 1
-    fi
-}
+        #yarn run test
```

Comment:
We should make an issue to add tests. In the past, we've commented out test execution in a STRTA and it went on to silently pass without executing tests for months.